cli/src/commands/lookup-table/initialize.ts

```diff
@@ -0,0 +1,32 @@
+import { Command } from "@oclif/core";
+import { CustomLoader, setAnchorProvider } from "../../utils";
+import { initLookUpTable, useWallet } from "@lightprotocol/zk.js";
+import { PathOrFileDescriptor, readFileSync } from "fs";
+import { Keypair } from "@solana/web3.js";
+
+class InitializeCommand extends Command {
+  static description = "Initialize new lookup table.";
+
+  static examples = ["light lookup-table:initialize"];
```

Comment:
please add a flag so that we can add additional accounts (you can add a flag with multiple = true)
This can be useful for relayers which will need to initialize their own lookuptables so that the lookuptable includes the relayer recipient